src/libxively/event_dispatcher/xi_event_handle_typedef.h

```diff
@@ -115,6 +115,11 @@ typedef struct xi_event_handle_s
     uint8_t target_tid;
 } xi_event_handle_t;
 
+#define xi_make_empty_event_handle( target_tid )                                         \
```

Comment:
I think you are right - but I treat those - function-like-macros - similar to normal functions - they are small enough to live as a function.